admission-controller/init/src/utils/logger.ts

```diff
@@ -0,0 +1,26 @@
+import { IncomingMessage } from 'http';
+import pino from 'pino';
+
+type FormattedLog = {
+  msg: string
+  error: string | undefined
+  errorDetails: any | undefined
+  responseStatus: number | undefined
+  responseMessage: string | undefined
+};
+
+const logger = pino({
+  name: 'Monokle:Init',
```

Comment:
Double checking whether `Monokle:Init` is what you want here?

Reply:
Yes, wanted to differentiate this logger (or rather logs from this logger) from the one from server container, so it can be filtered (in log aggregation scenarios). But maybe there is better way to do it?